### ISSUE TITLE
"Cannot read property 'length' of null"

### ISSUE THREAD

**Issue by rundis** [joshuafcole/claire#9]
I can't seem to get claire to work anymore. (LT 0.6.4). 

I just get the following in my console when trying to open (bound to ctrl+shift+f, not using emacs or vim keybindings).

``` javascript
Uncaught TypeError: Cannot read property 'length' of null
claire.js [104] claire.show
bootstrap.js [6188] b
bootstrap.js [6234] a
bootstrap.js [21050]    a
bootstrap.js [21054]    b
bootstrap.js [23123]    lt.objs.keyboard.trigger
bootstrap.js [23150]    lt.objs.keyboard.capture
bootstrap.js [23222]    anonymous
bootstrap.js [23223]    Mousetrap.handleKey
keyevents.js [562]  _handleKeyEvent
```

**Comment by joshuafcole** [comment]
Hmm, sounds like the #claire wrapper isnt getting inserted if I were to
guess. Actually inserting it is a pretty hacky process right now since I'm
wrestling with the console for the bottom bar. I've apparently already
fixed this on my development version while working on something else. I'll
get claire updated tonight. Thanks for the report!

**Comment by rundis** [comment]
Cool. I have a serious allergy towards Finder in OS/X. Claire looks promising in solving some of my file woes in LT :-)

**Comment by rundis** [comment]
BTW. It works when I open the console in a separate tab, but I guess you knew that already.

**Comment by joshuafcole** [comment]
Fix has been pushed as 0.1.7. It was a really simple bug due to switching
fro querySelectorAll (which returns an empty array when nothing is found)
to querySelector (which returns undefined) and not thinking to update my
existence check accordingly. Sorry for the delay! Please let me know if
problems persist.

Cheers,
Josh

On Wed, Mar 19, 2014 at 12:04 AM, Magnus Rundberget <
notifications@github.com> wrote:

> BTW. It works when I open the console in a separate tab, but I guess you
> knew that already.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/joshuafcole/claire/issues/9#issuecomment-38022940
> .

## 

Screw the environment. Please print this email immediately. And then burn
it.

**Comment by rundis** [comment]
Works on my machine ! Cheers.
